src/main/java/com/minecolonies/entity/ai/citizen/guard/AbstractEntityAIGuard.java

```diff
@@ -92,6 +135,23 @@
     private List<Entity> entityList;
 
     /**
+     * Positions of all items that have to be collected.
+     */
+    @Nullable
+    private List<BlockPos> items;
+
+    /**
+     * Number of ticks the guard is standing still
+     */
+    private              int   stillTicks              = 0;
+
+    /**
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) First sentence should end with a period. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=checkstyle%3AJavadoc_Style)